js/viz/chart.js

```diff
@@ -467,6 +467,13 @@ const dxChart = AdvancedChart.inherit({
         this._correctValueAxes(true);
     },
 
+    _setDeprecatedOptions() {
+        this.callBase();
+        _extend(this._deprecatedOptions, {
+            'argumentAxis.aggregateByCategory': { since: '23.1', message: 'Use the aggregation.enabled option instead' }
```

Comment:
1. We try to use `property` instead of `option`
2. Let's not write "instead" because "aggregation.enabled" is not a replacement
```suggestion
            'argumentAxis.aggregateByCategory': { since: '23.1', message: 'Use the aggregation.enabled property' }
```

Reply:
Fixed